addOns/soap/src/main/java/org/zaproxy/zap/extension/soap/ExtensionImportWSDL.java

```diff
@@ -164,6 +183,37 @@ public void fileUrlWSDLImport(final File file) {
         parser.extFileWSDLImport(file, THREAD_PREFIX + threadId++);
     }
 
+    private void addScript() throws IOException {
+        ExtensionScript extScript =
+                Control.getSingleton().getExtensionLoader().getExtension(ExtensionScript.class);
+        String scriptName = "SOAP Support.js";
```

Comment:
I'd suggest `SOAP_support.js` or even `soap_support.js`

Reply:
Okay. I went with `Soap Support.js` for consistency, because the GraphQL script is called `GraphQL Support.js`...

Reply:
Oh okay, stick with it then.